ui/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -31,13 +34,37 @@ export default class SendAssetRow extends Component {
 
   state = {
     isShowingDropdown: false,
+    sendableTokens: [],
   };
 
+  async componentDidMount() {
+    const sendableTokens = this.props.tokens.filter((token) => !token.isERC721);
+    this.setState({ sendableTokens });
+  }
+
   openDropdown = () => this.setState({ isShowingDropdown: true });
 
   closeDropdown = () => this.setState({ isShowingDropdown: false });
 
-  selectToken = (token) => {
+  clearUnsendableAssetError = () => {
+    this.props.setUnsendableAssetError(false);
+    this.props.updateSendErrors({ unsendableAssetError: null });
+  };
+
+  selectToken = async (token) => {
+    if (token && token.isERC721 === undefined) {
+      const updatedToken = await this.props.updateTokenType(token.address);
+      if (updatedToken.isERC721) {
+        const error = `Sending of collectible token ${token.symbol} is not currently supported`;
```

Comment:
We should localize this :)

Reply:
actually this error specificity isn't really needed. Its just a backend error that is never shown to the user. I'm going to remove and add generic error string.